web/admin/src/pages/document/component/ImportDoc/Confluence.tsx

```diff
@@ -95,37 +103,67 @@ const ImportDocConfluence = ({
     setCurrentFileIndex(0);
     try {
       for (let i = 0; i < acceptedFiles.length; i++) {
-        const pages = await postApiV1CrawlerConfluenceAnalysisExportFile({
+        const { id = '', docs = [] } = await postApiV1CrawlerConfluenceParse({
           file: acceptedFiles[i],
           kb_id,
         });
-        for (const page of pages) {
+        setId(id);
+        for (const page of docs) {
           setItems(prev => [
             {
-              url: page.id! as unknown as string,
+              url: page.id!,
               title: page.title!,
-              content: page.content!,
+              content: '',
```

Comment:
在处理Confluence导入时，content字段被设置为空字符串，但在后续流程中可能需要该字段的值。请确认是否需要从API获取实际内容或添加注释说明原因。

```suggestion
// 在这里添加从API获取content的逻辑，或者添加注释说明为什么content为空
content: '', // TODO: 获取实际内容或说明原因